tests/compiler/wasi/trace.ts

```diff
@@ -0,0 +1,5 @@
+import "wasi";
```

Comment:
Wooooaaaahhh this is soooo cool! :boom: :tada: 

Thank you so much for this, it's like 1000x cooler than what I was suggesting haha! :smile: